packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -511,8 +526,8 @@ - (void)mapView:(GMSMapView*)mapView didLongPressAtCoordinate:(CLLocationCoordin
 
 static NSDictionary* PointToJson(CGPoint point) {
   return @{
-    @"x" : @((int)point.x),
-    @"y" : @((int)point.y),
+    @"x" : @((int)(point.x + 0.5)),
```

Comment:
+ 0.5?

Reply:
This is to round to the closest int. 
The current code is getting a floor, which is less accurate.
updating to use `lroundf` to make it more self explanatory.